src/webgpu/capability_info.ts

```diff
@@ -12,6 +12,19 @@ import { ImageCopyType } from './util/texture/layout.js';
 
 // Base device limits can be found in constants.ts.
 
+// Adapters
```

Comment:
nit: maybe these should just be in the device_allocation.ts for now. Not clear where else they will be used (yet), and it's not exactly "capabilities" like much of the info in this file.

Reply:
Done, but we will probably want to eventually move them somewhere more common since I think they will be useful for future coverage testing.